### ISSUE TITLE
Switch N8N server

### ISSUE THREAD

**Issue by mkllnk** [openfoodfoundation/openfoodnetwork#12402]

**Comment by mkllnk** [comment]
@rioug What you think about pointing the old domain to the new server? Is it easy to connect n8n to two domains? Obviously, we have to activate all the need workflows first and can't do that just yet.

**Comment by rioug** [comment]
We need to make sure we can access prod DB from the new router before doing that as well.

Looks like it's fairly easy to connect n8n to two domain, it should be just a matter of configuring [traefik](https://doc.traefik.io/traefik/) (Edge router) to point the two domain to the n8n container, ie: https://frigi.ch/en/2022/07/multiple-domains-for-the-same-container-with-traefik/

And then updating the DNS record to point to the server.

**Comment by kirstenalarsen** [comment]
We can't complete the shutdown of the old server (which UK is paying for) until all the workflows on the old one have been transferred off it. Please look and attend to this @RachL @amidaOFN . . who else?

**Comment by RachL** [comment]
I'm really sorry it's taking me so long but I'm also anxious to break any prod workflow. It's my first time using n8n, so I also have to learn how to use it so I'm slow as hell

did any instance had to change something on the new server so far? If we are confident, perhaps we can migrate FR also without testing :/

**Comment by mkllnk** [comment]
> did any instance had to change something on the new server so far?

I don't think so. And I don't think that you can spot potential bugs beforehand. It's more about activating a workflow on the new server and check that it executed correctly. If it failed, you can always go back to the old one and ask for help.

**Comment by RachL** [comment]
Thanks Maikel! 

I have an error on one workflow (it's a subscription to a newsletter tool). It fails on the last step "import contacts in list" I've checked the credentials are ok.
Where else can I look?

![image](https://github.com/openfoodfoundation/openfoodnetwork/assets/6525576/bd6e2977-f623-40a2-9d77-2dd191a35ada)

**Comment by RachL** [comment]
@kirstenalarsen @mkllnk FYI also, before turning off the server, FR (and UK?) need to be monitored in NewRelic (currently it's done via Kuma, which is running thanks to cloudron on this UK server).

**Comment by rioug** [comment]
@RachL this is tracked here : https://github.com/openfoodfoundation/ofn-install/issues/926

**Comment by mkllnk** [comment]
@RachL 

> I have an error on one workflow (it's a subscription to a newsletter tool). It fails on the last step "import contacts in list" I've checked the credentials are ok.

How did you check the credentials? Did you copy the API key another time? The error message indicates that the API key isn't right.

**Comment by mkllnk** [comment]
I think that this is done. All workflows were migrated.
